client/CServerHandler.cpp

```diff
@@ -161,6 +161,18 @@ void CServerHandler::startLocalServerAndConnect()
 		threadRunLocalServer->join();
 
 	th->update();
+	
+	try
+	{
+		CConnection testConnection(settings["server"]["server"].String(), getDefaultPort(), NAME, uuid);
+		logNetwork->error("Port is busy, kill other vcmiserver processes");
```

Comment:
what if we connect and send special kill package so that server gets shut down?

Reply:
we need somehow to check if the server was started by this client. Otherwise we can kill remote server